logchange-core/src/main/java/dev/logchange/core/infrastructure/persistance/changelog/FileChangelogRepository.java

```diff
@@ -50,6 +52,25 @@ public Changelog find() {
         return Changelog.of(versions, archives);
     }
 
+    @Override
+    public Changelog findXML() {
+        List<File> inputFiles = getInputFiles();
+
+        List<ChangelogVersion> versions = new LinkedList<>();
+        List<ChangelogArchive> archives = new LinkedList<>();
+
+        for (File file : inputFiles) {
+            if (isVersionDirectory(file)) {
+                versions.add(getChangelogVersion(file));
+            }
+            if (isXmlArchive(file)) {
+                archives.add(getChangelogArchive(file));
```

Comment:
but from what i understand this method does not work now ?

Reply:
Currently it is identical to the findMarkdown method, I just seperated them for future use. I can change them back to how they were if you prefer?